products/emberjs.md

```diff
@@ -27,10 +27,17 @@ auto:
 # - support(x) = release(x+1)
 # - eol(x) = release(x+1)
 releases:
--   releaseCycle: "5.3"
-    releaseDate: 2023-09-18
+-   releaseCycle: "5.4"
+    releaseDate: 2023-10-30
     support: true
     eol: false
+    latest: "5.4.0"
+    latestReleaseDate: 2023-10-30
```

Comment:
```suggestion
    releaseDate: 2023-11-03
    support: true
    eol: false
    latest: "5.4.0"
    latestReleaseDate: 2023-11-03
```

I know this will be overwritten by the automation, but I think it's good to have it at least in the git history.